pkg/types/v2/file_existence.go

```diff
@@ -111,6 +112,34 @@ func (ft FileExistenceTest) Run(driver drivers.Driver) *types.TestResult {
 			result.Fail()
 		}
 	}
+	if ft.IsExecutableBy != "" {
+		perms := info.Mode()
+		switch ft.IsExecutableBy {
+		case "any":
+			if perms&0111 == 0 {
+				result.Errorf("%s has incorrect executable bit. Expected to be executable by any, Actual: %s", ft.Path, perms.String())
```

Comment:
this error message might be a bit confusing since it will combine the group identifier (e.g. `any`) with the actual permission bits. could you update it to extract the group ID from the permission bits?

Reply:
Eh yeah now that I think about it I guess this might be the best way to put it. In my mind I imagined something like

```
/usr/bin/foo has incorrect executable bit. Expected: `any`, Actual: `none`
```

```
/usr/bin/foo has incorrect executable bit. Expected: `group`, Actual: `all`
```

but it's probably more useful to just see the entire permission string.